app/src/main/java/org/simple/clinic/widgets/Views.kt

```diff
@@ -280,6 +281,24 @@ fun ViewPropertyAnimator.setDuration(duration: Duration): ViewPropertyAnimator {
   return setDuration(duration.toMillis())
 }
 
+fun View.increaseTappableArea(verticalSpace: Int, horizontalSpace: Int) {
+  val parent = parent as View
+
+  parent.post {
```

Comment:
This assumes that the `View` has already been laid out and bounds are available when invoking this method. Should we consider using the `executeOnMeasure` extension instead of the `post` method?

Reply:
Umm, usually by the time `post` is called the view is laid out right? So it should be fine.

Reply:
Have we tested this change?

Reply:
Yep. It works ✅, you can try it out but tapping at bottom of the right bottom and it should show the ripple for card button.